docs/commands.md

```diff
@@ -50,6 +51,21 @@ This command retrieves a set of `TimeEntry` instances whose time period (time co
  - business failure - see [Record Time](https://collaboration.homeoffice.gov.uk/jira/browse/EAHW-925) (access required) for business failure scenarios. Also see [standard command output](TODO) (**TODO**)  for how to report business failures
  - technical failure - see [standard command output](TODO) (**TODO**) for how to report technical failures
 
+## add FlexChange
+This command causes a new `FlexChange` to be created. The creation of a `FlexChange` is driven by a change to an existing `TimeEntry` (change to the `TimeEntry.actualStartTime`and/or the `TimeEntry.actualEndTime` values). Without this modification the `FlexChange` cannot exist. On this basis it is crucial that the implementation of this command ensures that the `TimeEntry` modification and the `FlexChange` creation are committed as an atomic unit. If either operation fails then the whole command should fail and any persisted changes should be rolled back to the state that the entities were in prior to the command being invoked. 
```

Comment:
Does`TimeEntry.actualStartTime` here want to be updated to reflect the new field names for the times?